usaspending_api/download/v2/download_column_historical_lookups.py

```diff
@@ -1693,35 +1777,119 @@
                 ("owning_agency_name", "treasury_account__federal_account__parent_toptier_agency__name"),
                 ("reporting_agency_name", "reporting_agency_name"),  # Column is annotated in account_download.py
                 ("submission_period", "submission_period"),  # Column is annotated in account_download.py
-                ("federal_account_symbol", "treasury_account__federal_account__federal_account_code"),
-                ("federal_account_name", "treasury_account__federal_account__account_title"),
                 ("agency_identifier_name", "agency_identifier_name"),
                 ("budget_function", "budget_function"),  # Column is annotated in account_download.py
                 ("budget_subfunction", "budget_subfunction"),  # Column is annotated in account_download.py
+                ("federal_account_symbol", "treasury_account__federal_account__federal_account_code"),
+                ("federal_account_name", "treasury_account__federal_account__account_title"),
                 ("program_activity_code", "program_activity__program_activity_code"),
                 ("program_activity_name", "program_activity__program_activity_name"),
                 ("object_class_code", "object_class__object_class"),
                 ("object_class_name", "object_class__object_class_name"),
                 ("direct_or_reimbursable_funding_source", "object_class__direct_reimbursable"),
                 ("disaster_emergency_fund_code", "disaster_emergency_fund__code"),
                 ("disaster_emergency_fund_name", "disaster_emergency_fund__title"),
-                ("obligations_incurred", "obligations_incurred"),
+                ("obligations_incurred", "obligations_incurred_by_program_object_class_cpe"),
```

Comment:
Looking through account_download.py I don't think this change will even get picked up because it is getting set to the value held by the `treasury_account` dictionary above. 

Can you speak to why you changed this?

Reply:
This was a mistake from copying the new fields from `treasury_account` to `federal_account`. It has been reverted.